tests/unit/cli_commands/test_chat_stats.py

```
@@ -0,0 +1,183 @@
+    assert data.reasoning == "thinking"
+
+
+def test_make_response_data_content_only_is_text() -> None:
+    assert make_response_data("answer", None) == TextResponseData(text="answer")
+
+
+def test_make_response_data_empty_is_none() -> None:
+    assert make_response_data(None, None) is None
+
+
+def _record(start_ns, perf_nss, *, output, reasoning=None, input=None, usage=None):
```

Ruff (0.15.20)

[error] Function argument `input` is shadowing a Python builtin

(A002)